x-pack/plugins/ux/public/components/app/rum_dashboard/visitor_breakdown_map/embedded_map.tsx

```diff
@@ -65,6 +65,7 @@ export function EmbeddedMapComponent() {
   const factory = embeddablePlugin.getEmbeddableFactory(MAP_SAVED_OBJECT_TYPE);
 
   const input: MapEmbeddableInput = {
+    embeddingApp: 'ux',
```

Comment:
The `ux` plugin does not appear to have a const for the plugin name, so hardcoding it here.